pkg/daemon/daemon.go

```diff
@@ -419,6 +455,11 @@ func (d *Daemon) Wait() error {
 		// nydus-snapshotter, p.Wait() will return err, so here should exclude this case
 		if _, err = p.Wait(); err != nil && !errors.Is(err, syscall.ECHILD) {
 			log.L.Errorf("failed to process wait, %v", err)
+
+		} else {
+			// FIXME: Snapshotter can't wait for non-child process that exits and finishes unmounting.
+			// So mount point may still be residual. As a workaround, wait for some time.
+			time.Sleep(time.Millisecond * 500)
```

Comment:
It's so hacky, why not check daemon sock/mountpoint?

Reply:
We can try busy check for the mountpoint until it is umounted.

Reply:
fixed